src/mrack/transformers/aws.py

```diff
@@ -36,9 +36,9 @@ class AWSTransformer(Transformer):
 
     async def init_provider(self):
         """Initialize associate provider."""
-        images = self.config["images"].values()
+        images = self.config["images"]
         await self._provider.init(
-            image_names=images,
+            images=images,
```

Comment:
We have `distros`, `image` and `image_names`, one for each provider. right? :/

Reply:
Hmmmm i can try to do something with it.

Reply:
Apparently, not all of them are dicts.